frontends/web/src/utils/transport.common.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2022 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export type TQueryPromiseMap = {
+  [key: number]: {
+    resolve: (value: unknown) => void,
+    reject: (value: unknown) => void,
+  };
+};
+
+export type TMsgCallback = (payload: any) => void; // should be { type: any; data: any; meta: any; } or
```

Comment:
Temporary moved payload and msgCallback type here to share, typed this in the next PR, so all apiWebsocket return either event or event-legacy type.

Reply:
The next PR https://github.com/digitalbitbox/bitbox-wallet-app/pull/1775